gremlin-core/src/main/java/org/apache/tinkerpop/gremlin/process/traversal/step/sideEffect/AggregateLocalStep.java

```diff
@@ -39,7 +39,9 @@
 
 /**
  * @author Marko A. Rodriguez (http://markorodriguez.com)
+ * @deprecated As of release 3.7.5, use local() with aggregate() instead
```

Comment:
a bit of a nit i gues, but I think in this case, the deprecation comment is:

> @deprecated As of release 3.7.5, not replaced

As a Gremlin step implementation, it's just gone...there's no direct replacement for usage. `GraphTraversal` and `__` as user facing get the slightly fancier message.